etc/DependencyInstaller.sh

```diff
@@ -314,6 +314,10 @@ _installOrTools() {
         if [[ $osVersion == rodete ]]; then
             osVersion=11
         fi
+        if [[ $os == ubuntu && $osVersion == 25.04 ]]; then
+            # FIXME make do with 24.04 for now until official release for 24.04 is available
```

Comment:
This doesn't make sense with 24.04 repeated twice.

Reply:
fixed